src/client/components/email-form/readme.md

```diff
@@ -0,0 +1,16 @@
+# Email form
```

Comment:
Nice readme! I would still link to Formspree and explain that we use its `_next`, `_subject`, `_gotcha` and `_language` features, so other devs understand where these things come from. Maybe even an extra comment in the code next to it?

Reply:
Yes good one.